lib/pry/cli.rb

```diff
@@ -140,6 +148,11 @@ def start(opts)
     Pry.config.exec_string += input
   end
 
+  on "no-correct-indent", "Do not automatically correct indentation errors emitted by " \
```

Comment:
I don't think we need to introduce this new flag. It's unlikely this will be used by anybody. Since this seem to exist just for the sake of making tests pass, I'd try to find another way.

Reply:
Sure. I will add some sort of fix to the test. It's a bit weird that Pry writes silent junk to stdout in this scenario though.